easybuild/easyconfigs/f/foss_mv2/foss_mv2-2016a.eb

```diff
@@ -0,0 +1,35 @@
+easyblock = 'Toolchain'
+
+name = 'foss_mv2'
+version = '2016a'
```

Comment:
the `<year>[ab]` naming is kind of reserved for the "common toolchains" (`foss` and `intel`) which we redefine twice a year (Jan/Jul)

since we now have a pretty meaningless versioning of `gmvolf`, maybe we can switch to `<year>.<month>` there too, like we do for additional definitions of `intel` and `foss` that are not considered 'common'

This is not perfect either though, since you could easily define two variants of `gmvolf` in the same month (one using GCC v4, the other using GCC v5 for example)...

I'm open to suggestions here...